src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -130,13 +130,13 @@ Type 'help' to get help.</value>
   <data name="PreviewUpdateNotificationMessage" xml:space="preserve">
     <value>  {1} A new PowerShell preview release is available: v{0} {2}
   {1} Upgrade now, or check out the release page at:{3}{2}
```

Comment:
"Upgrade now, or check ..." - I understand this as "Press the button to upgrade now or read the online page"

Reply:
Talked to @joeyaiello about this, we'll keep the language as-is for now as on Linux and macOS (via homebrew) users should understand easily what `upgrade` means.  Windows users still don't have this option.  We can adjust the language here later.